tests/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ set_target_properties(
   CXX_STANDARD_REQUIRED ON
   CXX_EXTENSIONS OFF
   COMPILE_FLAGS "${WARNING_FLAGS}"
-  UNITY_BUILD ON
+  UNITY_BUILD ${USE_UNITY_BUILD}
```

Comment:
Just curious - is there no way to set all these target_properties in a common location and then inherit? Not a huge deal of course just learning.

Reply:
I dug through the documentation and couldn't find a way to set it globally